lab_1_classify_by_unigrams/main.py

```diff
@@ -3,31 +3,34 @@
 Language detection
 """
 
-
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+    if isinstance(text, str):
```

Comment:
Consider escaping the function immediately after discovering that the value is incorrect. It makes the code look more readable.

Reply:
done